### PR TITLE
Only do peerDep check if npm < 3.0

### LINKED ISSUE #29: cldr-data install.js is broken wrt. peer dependencies
It has three problems:

1. it fails, but then exits with status `0`: https://github.com/rxaviers/cldr-data-npm/blob/master/install.js#L52, this is incorrect. Failing scripts must exit with a non-`0` status to indicate failure.
2. its determination of whether it is installing because of a peer dependency appears to be a protection to be future proof with npm3... but actually, it fails with npm3
3. the message is not grammatically correct English, but that's pretty minor

Our scenario is this:

- ourpackage
  - depends on strong-globalize, a DIRECT normal npm dependency
    - strong-globalize direct depends on cldr-data
    - strong-globalize direct depends on globalize
      - globalize peer depends on cldr-data

This is all correct. cldr-data is installed via direct dependency, NOT via the globalize peerDependency, but cldr-data doesn't download, and after printing a message, reports success via its exit status.

cldr-data's gets installed by npm3 at the top-level, using the flattening, and this appears to confuse the script.

### PR DESCRIPTION
**Pull request by marcelklehr**
fixes #29

### CODE REVIEW

**Comment by rxaviers** [comment]
Thank you for the fix, please could you read [CLA]() and sign up your commit by `git commit --amend -s`?

**Comment by rxaviers** [comment]
Just let me know on any problem please.

File: install.js [hunk 1]
```diff
@@ -13,9 +13,18 @@ var coverage, parentPackage, peerPackages, srcUrl;
...
+var child_process = require('child_process');
 
 var options = {};
 
+var isNpm3
+try {
+  var npmv = child_process.execSync('npm -v').toString('utf8');
+  isNpm3 = (npmv.split('.')[0] == '3');
+}catch(){
+  isNpm3 = true // better safe than sorry
```
Comment (rxaviers) [inline install.js]: Missing `;`.

Comment (rxaviers) [inline install.js]: Please can you:

```diff
- isNpm3 = true // better safe than sorry
+ // Better safe than sorry.
+ isNpm3 = true;
```

**Comment by rxaviers** [comment]
Ref: #23

**Comment by marcelklehr** [comment]
done :)

**Comment by rxaviers** [comment]
Landed thanks. Released as new patch fix.

**Pull request closed without merging**